setup.py

```diff
@@ -19,41 +19,49 @@
 """
 
 from setuptools import setup, find_packages
-import sys, os, io
+import io
+
 
 def readfile(filename, split=False):
     with io.open(filename, encoding="utf-8") as stream:
         if split:
             return stream.read().split("\n")
         return stream.read()
 
-readme = readfile("README.md", split=True)[3:]  # skip title
-requires = readfile("requirements.txt", split=True)
-license = readfile("LICENSE")
+
+package_readme = readfile("README.md", split=True)[3:]  # skip title
+package_license = readfile("LICENSE")
+package_dependencies = [
+    "transforms3d",
+    "setuptools",
+    "PySide2",
+    "numpy",
+    "gias2",
+    "opensim >= 4.2"
+]
 package_data = {
-  'mapclientplugins.fieldworkgait2392geomstep': [
-    'data/*',
-  ],
+    'mapclientplugins.fieldworkgait2392geomstep': [
+        'data/*',
+    ],
 }
 
-
 setup(name=u'mapclientplugins.fieldworkgait2392geomstep',
-    version='0.1.3',
-    description='',
-    long_description='\n'.join(readme) + license,
-    classifiers=[
-        "Development Status :: 3 - Alpha",
-        "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: Python",
-    ],
-    author=u'Ju Zhang',
-    author_email='',
-    url='https://github.com/mapclient-plugins/fieldworkgait2392geomstep',
-    license='APACHE',
-    packages=find_packages(exclude=['ez_setup',]),
-    namespace_packages=['mapclientplugins'],
-    include_package_data=True,
-    package_data=package_data,
-    zip_safe=False,
-    install_requires=requires,
-    )
+      version='0.1.3',
```

Comment:
Remind me to change this to scrape the version number from the plugin `__init__.py` file.

Reply:
I have also made an attempt at this one. Seems to work.